base/skeleton/gradle-build/build.gradle

```diff
@@ -11,18 +11,16 @@ repositories {
 @repositories@
 }
 
-dependencyManagement {
-    imports {
-        mavenBom 'io.micronaut:micronaut-bom:@version@'
-    }
-}
-
 configurations {
     // for dependencies that are needed for development only
     developmentOnly 
 }
 
 dependencies {
+    implementation enforcedPlatform("io.micronaut:micronaut-bom:$micronautVersion")
+    compileOnly enforcedPlatform("io.micronaut:micronaut-bom:$micronautVersion")
+    annotationProcessor enforcedPlatform("io.micronaut:micronaut-bom:$micronautVersion")
```

Comment:
Think it will probably needs to be in `testAnnotationProcessor` scope too

Reply:
Done.